dms/models/dms_file.py

```diff
@@ -644,7 +644,12 @@ def _inverse_content(self):
 
     def _create_model_attachment(self, vals):
         res_vals = vals.copy()
-        directory = self.env["dms.directory"].sudo().browse(res_vals["directory_id"])
+        directory = False
+        directory_model = self.env["dms.directory"].sudo()
```

Comment:
I'm not sure sudo is a good idea here. You don't want your files to be uploaded to folders where you don't have permissions, right? 🤔

Reply:
There was previously a `sudo`, maybe for the attachment thing. Although in general is not good idea to require `sudo` (and more in v12 that changes the uid), do you think this removal can provoke any error?

Reply:
I tested functionally after this removal and it seems to work fine 👍🏼